test/javascripts/acceptance/chat-keyboard-shortcuts-test.js

```diff
@@ -251,4 +251,28 @@ acceptance("Discourse Chat - Keyboard shortcuts", function (needs) {
       "modal dismissed after submitting link"
     );
   });
+
+  test("Dash key (-) opens chat float", async function (assert) {
+    await visit("/latest");
+    this.chatService.set("sidebarActive", false);
+    this.chatService.set("chatWindowFullPage", false);
+
+    await triggerKeyEvent(document.body, "keydown", "-");
+    await settled();
+    assert.ok(exists(".topic-chat-drawer-content"), "chat float is open");
+  });
+
+  test("Escape to close chat float", async function (assert) {
+    await visit("/latest");
+    this.chatService.set("sidebarActive", false);
+    this.chatService.set("chatWindowFullPage", false);
+
+    await click(".header-dropdown-toggle.open-chat");
+    await settled();
+
+    const composerInput = query(".chat-composer-input");
+    await focus(composerInput);
+    await triggerKeyEvent(document.body, "keydown", "Escape");
```

Comment:
I think instead of triggering the key event on `document.body`, it has to do `triggerKeyEvent(composerInput)` since that needs to be focused and receiving the key event to have this shortcut work

Reply:
Thanks! This fixed it 😄 Tests are passing now